### ISSUE TITLE
Allow anonymous user to generate an application via user interface

### ISSUE THREAD

**Issue by vishal423** [jhipster/jhipster-online#162]
Should be done after #160

**Comment by SudharakaP** [comment]
Since #160 is done I am closing this.

**Comment by vishal423** [comment]
It was meant to be worked upon after the merge :)

**Comment by SudharakaP** [comment]
@vishal423 : Ops sorry. I think I closed this an another one with the idea the changes are already incorporated in the update.

**Comment by swarajsaaj** [comment]
I feel this would be very nice to have. @SudharakaP @vishal423 Shall I pick this up if no one is working on it?

**Comment by vishal423** [comment]
Feel free to pick up @swarajsaaj

**Comment by vishal423** [comment]
@deepu105 @pascalgrimaud @mraible  - It seems a good candidate to have a bounty. Can you add one ?

**Comment by swarajsaaj** [comment]
Thanks @vishal423 ,Shall we also move this to homepage once we make it for anonymous users (like https://start.spring.io/, https://code.quarkus.io/ or https://code.quarkus.io, maybe with a refactored interface too).

**Comment by pascalgrimaud** [comment]
@vishal423 : done

**Comment by SudharakaP** [comment]
> Thanks @vishal423 ,Shall we also move this to homepage once we make it for anonymous users (like https://start.spring.io/, https://code.quarkus.io/ or https://code.quarkus.io, maybe with a refactored interface too). Any views on this @pascalgrimaud @vishal423 @SudharakaP ?

Thanks for taking it up @swarajsaaj. My suggestion is that as an MVP first let us just open it up for anonymous users. Changing the interface can be done on a separate issue. 😄

**Comment by swarajsaaj** [comment]
bounty claimed https://opencollective.com/generator-jhipster/expenses/42422

**Comment by vishal423** [comment]
Can we add a section / link on the main page to navigate to the UI interface to allow anonymous users to generate applications?

**Comment by swarajsaaj** [comment]
@vishal423 The "Create Application" link in the left navigation bar is now available for anonymous users. Do you mean something on the right section too?

**Comment by vishal423** [comment]
@swarajsaaj thanks for the confirmation. It seems a caching issue on my side. I can see the option now and looks good to me.
